ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -3137,7 +3137,7 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
                     break;
                 case StrategyTradeWorkFlowState.ExitTradeClosePositionsPending:
                     //execution event or realtime event or timer moves event on
-                    if (IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode) break;

```

Comment:
This is required - IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode use the same paths a lot of the time-